### PR TITLE
DM-39178: Add deterministic seed to ks-test model sampling.

### PR DESCRIPTION
**Pull request by erykoff**

### CODE REVIEW

File: python/lsst/cp/pipe/ptc/cpExtractPtcTask.py [hunk 1]
```diff
@@ -1009,7 +1009,15 @@ def computeGaussianHistogramParameters(self, im1Area, im2Area, imStatsCtrl, mu1,
             sigmaFit = out.params["sigma"].value
 
             # Calculate KS test p-value for the fit.
-            gSample = scipy.stats.norm.rvs(size=numOk, scale=sigmaFit, loc=out.params["center"].value)
+            # Seed this with the mean value, so that the same data will get the
+            # same result.
+            randomSeed = int((mu1 + mu2)/2.)
```
Comment (mfisherlevine) [inline python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:1014]: It's an interesting thought. But is it really what you want? Upstream changes which cause changes in the inputs will cause a seed-change - why not just hard code it like we do everywhere else? I mean, I think it's fine, just kinda curious if you really do want to do that...

**Review by mfisherlevine — APPROVED** [review]

File: python/lsst/cp/pipe/ptc/cpExtractPtcTask.py (hunk 1 above)
Comment (erykoff) [reply python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:1014]: I don't know if I fully believe in completely hard-coding seeds outside tests; in fact we often use exp number or something that changes but is deterministic.  I just wanted something quick that would give the same output for the same input.

Comment (mfisherlevine) [reply python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:1014]: Ah, I'd forgotten that it was tests I was thinking of. OK, nice, thanks.

Comment (mfisherlevine) [reply python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:1014]: The fact the fit is so on the hairy edge that seeds cause it to fail is a different worry though, but for now, fixing random Jenkins failures is clearly more important.

Comment (erykoff) [reply python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:1014]: Well, the hairy-edge part is going to be fixed on DM-39167.

Comment (mfisherlevine) [reply python/lsst/cp/pipe/ptc/cpExtractPtcTask.py:1014]: OK, I figured, thanks 🙂

**Pull request merged**